### PR TITLE
Add missing translations: es

### PR DESCRIPTION
**Pull request by sorja**

### CODE REVIEW

**Review by minotogna — COMMENTED** [review]
Any reason why it's still in draft ?

**Comment by sorja** [comment]
> Any reason why it's still in draft ?

Asking Xavi to glance over it before merging

File: src/i18n/resources/es.js [hunk 1]
```diff
@@ -523,6 +554,8 @@ El equipo de FRA
     ndpMissingValues: 'El punto de datos nacionales tiene valores faltantes',
```
Comment (xavidelamo) [inline src/i18n/resources/es.js]: Please provide the original English statement so I can translate it properly

File: src/i18n/resources/es.js [hunk 2]
```diff
@@ -932,6 +969,22 @@ El equipo de FRA
     valueMustBePositive: 'El valor debe ser superior a cero',
     emptyField: 'Este campo está vacío',
     mustBeEqualToTotalGrowingStock: 'El valor debe ser igual al Existencias totales en formación (2a)',
```
Comment (xavidelamo) [inline src/i18n/resources/es.js]: Please replace by ''El valor debe ser igual al valor total de Existencias Forestales en Formación(2a)

File: src/i18n/resources/es.js (hunk 1 above)
Comment (sorja) [reply src/i18n/resources/es.js]: ndpMissingValues: 'National data point has missing values',

Comment (xavidelamo) [reply src/i18n/resources/es.js]: Please replace by : "El punto de dato nacional tiene valores faltantes"

**Pull request merged**